miageWebApp/src/app/modal/modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.scss']
+})
+export class ModalComponent implements OnInit {
+  teacherForm: FormGroup;
+  showModal: boolean;
```

Comment:
showModal must be an @input and change from the parent component

Reply:
in progress